### ISSUE TITLE
Custom Messages On Alert Descriptions

### ISSUE THREAD

**Issue by roomit-xyz** [SigNoz/signoz#4793]
This is alert based on metrics logs.

```
💔 ALERT
Rule: sudo execution | The rule threshold is set to 3, and the observed metric value is 6
```

I want ask, Is possible custom message?
due I have change alert description, message still like that.

Best Reagars

**Comment by srikanthccv** [comment]
Yes, you can update the description. What have changed the alert description to? Was it successfully updated when saved?

**Comment by roomit-xyz** [comment]
Hallo Team,

I have tried test change description, but the message always show like that. we used webhook channel. is possible, the message desription text cut by webhook matrix?

Is true config description in here?
![Screenshot-45](https://github.com/SigNoz/signoz/assets/1472909/f97c7a41-d845-408c-aab4-391a1f1f3180)


Best Regards
Dwi
